Update retreat page after CAKE meeting (#100)
CAKE-DRI/CAKE-DRI.github.io: _pages/activities/KE-retreats.md

---
title: Retreats
permalink: /about/retreats/
layout: no-title-page
---

<br>

<div style="display: flex; justify-content: center; gap: 10px;">
  <img src="/assets/images/ke_retreats.png" width="90%">
</div>

<br>

> The CAKE Retreats programme provides funding for community-led retreats to shape the future of the UK's digital research infrastructure and large-scale computing ecosystem.

Reviewer: ```suggestion
The CAKE Retreats programme provides funding for community-led retreats to shape the future of the UK's Digital Research Infrastructure (DRI) and large-scale computing ecosystem.
```

> The programme supports the organisation of retreats that create dedicated time and space for focused collaboration, enabling communities to develop new ideas, build partnerships, identify strategic priorities, and produce shared outputs such as white papers, reports, or journal special issues. We encourage participation and engagement across academia, industry, and the wider research and innovation landscape. We welcome proposals from the community for retreat themes that address important challenges, opportunities, and priorities for the UK digital research infrastructure (DRI) ecosystem.

Reviewer: ```suggestion
The programme supports the organisation of retreats that create dedicated time and space for focused collaboration, enabling communities to develop new ideas, build partnerships, identify strategic priorities, and produce shared outputs such as white papers, reports, or journal special issues. We encourage participation and engagement across academia, industry, and the wider research and innovation landscape. 

We welcome proposals from the community for themed retreats that address important challenges, opportunities, and priorities for the UK DRI ecosystem.
```

## Application information

We are committed to fostering a diverse and inclusive environment that reflects the breadth of the CAKE network and the wider UK Digital Research Infrastructure community. We actively encourage applications from all members of the community, regardless of background, discipline, or career stage. 

All participants in the CAKE network are expected to follow the [CAKE Code of Conduct](https://www.cake.ac.uk/CAKEbox/code-of-conduct/), ensuring a supportive and respectful environment for everyone involved.

> We invite the community to propose retreat themes and topics that they believe are important, timely and beneficial to the UK digital research infrastructure community. These retreats should create opportunities for community building, collaboration and the creation of shared outputs and strategic priorities.

Reviewer: I'm not sure if it's 100% clear that we are asking people to organise a retreat, not just suggest themes/topics? Might be worth re-wording this slightly!

Reviewer 2: Nice catch ;-) I rephrased it to 'We invite the community to propose, organise and deliver retreats on themes and topics that they believe are important, timely and beneficial to the UK digital research infrastructure community.'

>  In total, there will be 5 themed retreats, each lasting 3-4 days. Total funding for retreats is £125,000.

Reviewer: Do we want to advertise how much money in total?

Author: Yep, happy to remove that!

<div style="text-align: center; margin-top: 2rem;">

>   <a href="{{ '[https://forms.gle/qe5jCLPkBJ9tGNKg6](https://docs.google.com/forms/d/e/1FAIpQLSc-NjBIF7YXGjzUX1KMh__8TtZx_u-eM8DzZNVZp2-ixmffww/viewform?usp=dialog)' | relative_url }}" class="btn btn--secondary btn--x-large">Apply here!</a>

Reviewer: For me, this didn't work locally without this change: 

```suggestion
  <a href="{{ 'https://docs.google.com/forms/d/e/1FAIpQLSc-NjBIF7YXGjzUX1KMh__8TtZx_u-eM8DzZNVZp2-ixmffww/viewform?usp=dialog' | relative_url }}" class="btn btn--secondary btn--x-large">Apply here!</a>
```

## EDI
Considerations around equality, diversity, inclusion, and accessibility are central to ensuring these activities are open to all. All applications will be reviewed in line with the network’s EDI policy, supported by best-practice guidance for reviewers to ensure fair and inclusive assessment. We have ambitions for specific targets, including aiming for around 50% of participants to come from non-academic organisations. In addition, all retreats will be delivered in a hybrid format to support broader participation and reduce barriers to engagement.

Please check our [Equality, Diversity, Inclusion & Accessibility](https://www.cake.ac.uk/CAKEbox/edia/) page on our [CAKEbox](https://www.cake.ac.uk/CAKEbox/)

## Timeline 
The programme runs from October 2025 through to April 2028, at which time all retreats must be complete and expenses claimed.

## Process 
Applications are open with a rolling deadline.

Awards of retreats will be administered through a robust and transparent competitive process overseen by the Review Committee (RC). Please check our [Submissions & Reviewing](https://www.cake.ac.uk/CAKEbox/submissions-and-reviewing/) page on the CAKEbox.

> Retreat themes will be proposed by the community as part of the application process. These applications should describe the proposed theme, its relevance for the UK DRI landscape and the anticipated outcomes and benefits for the wider community

Reviewer: How do they do this? The linked form seems to be for people wanting to organise a retreat themselves?

Author: So I thought people were applying to host a retreat by suggesting their own topics. I didn’t realise we were considering the idea of having one person propose a theme and someone else host that retreat - other than specific topics that we want to propose as CAKE

Reviewer: I agree! I think it would be great to just get retreat proposals rather than retreat topics, but maybe we could add a link to our email to get in touch if you have an idea but are struggling to define it?
